### PR TITLE
release artifacts for release v0.0.1

### PR DESCRIPTION
**Pull request by jaypipes**
Add Helm charts for S3, SNS, ECR, API GW v2, Elasticache, Step Functions
and DynamoDB service controllers.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jaypipes** [comment]
Need to rebase after pulling in fixes for service accounts from @A-Hilaly

**Review by A-Hilaly — COMMENTED** [review]
LGTM! 
nit: after #450/#476 we can also generate SFN helm charts :)

File: services/apigatewayv2/helm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Default values for ack-apigatewayv2-controller.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: amazon/aws-controllers-k8s
+  tag: apigatewayv2-v0.1.0
```
Comment (A-Hilaly) [inline services/apigatewayv2/helm/values.yaml:7]: s/v0.1.0/v0.0.1

**Comment by A-Hilaly** [comment]
just tested with multiple service controllers; it looks like there is a conflict with `ServiceAccount` default names. Sending a fix soon

File: services/apigatewayv2/helm/values.yaml (hunk 1 above)
Comment (jaypipes) [reply services/apigatewayv2/helm/values.yaml:7]: yikes... this is bad mistake on my part. will rebase and fix.

**Review by A-Hilaly — APPROVED** [review]
LGTM 👍

**Pull request merged**